R/dependencies.R

```diff
@@ -23,3 +23,30 @@ find_dependencies <- function(expr) {
        variables = unique(variables$get()))
 
 }
+
+
```

Comment:
So here, deps is a list of named vectors;
each vector has the name of a var, and the value is a vector of vars that it depends on.
deps is produced by parse_system_depends